Cargo.toml

```diff
@@ -5,3 +5,4 @@ resolver = "2"
 [profile.release]
 lto = true
 codegen-units = 1
+strip = "symbols"
```

Comment:
The docs (https://doc.rust-lang.org/cargo/reference/profiles.html) list the options, but don't really explain the difference between "debuginfo" and "symbols" :-( 
So, I had to go here: https://users.rust-lang.org/t/difference-between-strip-symbols-and-strip-debuginfo/96602 and learn it...thanks!

Reply:
I actually didn't mean to commit that change 🤦 
Let me undo that!